pytrip/vdx.py

```diff
@@ -548,22 +548,34 @@ def create_sphere(cube, name, center, radius):
     :returns: A new Voi object.
     """
     v = Voi(name, cube)
-    t = np.linspace(start=0, stop=2.0 * pi, num=100)
+    t = np.linspace(start=0, stop=2.0 * pi, num=100)  # num: sets the number of corners in sphere per slice.
     p = list(zip(np.cos(t), np.sin(t)))
+    del p[-1]  # delete the last point will be repeated later in add_contour anyway.
```

Comment:
I don't like using `del` -> it looks like C++ code, why not making following assignment:

```
p = list(zip(np.cos(t), np.sin(t)))[:-1]
```

Or even better:
```
t = np.linspace(start=0, stop=2.0 * pi, num=99, endpoint=False)
p = list(zip(np.cos(t), np.sin(t)))
```

Numpy manual (https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.linspace.html) says: 

>> endpoint : bool, optional
>> If True, stop is the last sample. Otherwise, it is not included. Default is True.

Reply:
ok!